src/styles/theme.js

```diff
@@ -1,4 +1,14 @@
-export const PRIMARY = '#E91E63';
-export const PRIMARY_DARK = '#710d2f';
-export const PRIMARY_LIGHT = '#ff6297';
-export const TEXT = '#fff';
\ No newline at end of file
+export const UNDERLAY_COLOR = '#E6E6E6';
+
+export const BLUE = '#4F6D7A';
+export const ORANGE = '#D57A66';
+export const GREEN = '#00BD9D';
+export const PURPLE = '#9E768F';
+
+export default {
+  white: '#FFFFFF',
+  border: '#E2E2E2',
+  inputText: '#797979',
+  lightGrey: '#F0F0F0',
+  darkText: '#343434',
```

Comment:
Values for all of these colors need to be imported from `colors.js` - no hardcoding